src/FSharp.Data.GraphQL.Samples.GiraffeServer/Schema.fs

```diff
@@ -140,40 +142,37 @@ module Schema =
             name = "Human",
             description = "A humanoid creature in the Star Wars universe.",
             isTypeOf = (fun o -> o :? Human),
-            fieldsFn = fun () -> 
+            fieldsFn = fun () ->
             [
                 Define.Field("id", String, "The id of the human.", fun _ h -> h.Id)
                 Define.Field("name", Nullable String, "The name of the human.", fun _ h -> h.Name)
                 Define.Field("friends", ListOf (Nullable CharacterType), "The friends of the human, or an empty list if they have none.",
-                    fun _ h -> 
-                        h.Friends
-                        |> List.map getCharacter 
-                        |> List.toSeq)
+                    fun _ (h : Human) -> h.Friends |> List.map getCharacter |> List.toSeq).WithQueryWeight(0.5)
```

Comment:
I added the query weight threshold middleware in the giraffe sample project. Friends now have a weight of 0.5 each, and the maximum weight is 5.0.